dlpy/lr_scheduler.py

```diff
@@ -181,6 +185,7 @@ def __init__(self, learning_rate, power):
 class ReduceLROnPlateau(FCMPLR):
     """
     Reduce learning rate on plateau learning rate scheduler
+    Reduce learning rate when loss has stopped improving for a patience number of epochs.
```

Comment:
for certain number of epochs (I am assuming this is set by a parameter or you decide right)?

Reply:
yes, correct

Reply:
would you please update the comment. thanks.